### ISSUE TITLE
Show docker with simultaneous left and right mouse click

### ISSUE THREAD

**Issue by MaikoID** [micheleg/dash-to-dock#582]
Hi, I would like to ask this feature where I  can invoke the docker when I press, simultaneous, left and right mouse click.

Similar as the default <Super>q to "in use keyboard shortcuts to active the apps"

What do you think?

**Comment by micheleg** [comment]
Thanks for the idea:
1. Technical issue: this would require to intercept all mouse click events, regardless of which is the current target, which I don't think it is possible
2. What is the idea? Isn't the active edge enough to show the dock with the mouse?

**Comment by MaikoID** [comment]
Yeah, it appears to be difficulty to implement maybe another project with only this purpose, handling mouse shortcuts, would be better. Either way the Docker would need changes to listen to this kind of request.

I really don't know if it will increase my productivity because I've never seen any similar behavior in others dockers. But I guess it will. Sometimes is good to see the docker when I want to without moving the mouse too much or use my left hand, a common issue I have because my wide resolution. What comes to my mind is if the docker could appear in either side of my screen after I push the respective edge, will be nice and a good replacement.
